### ISSUE TITLE
windows release

### ISSUE THREAD

**Issue by explodus** [bblanchon/pdfium-binaries#37]
there is no windows release in the last merge

**Comment by explodus** [comment]
https://github.com/bblanchon/pdfium-binaries/releases

**Comment by mara004** [comment]
I am also puzzled by this. I can't make a new release for PyPDFium unless there is a Windows binary, too.

**Comment by bblanchon** [comment]
The [build is failing](https://ci.appveyor.com/project/bblanchon/pdfium-binaries/build/job/ho9ega32y3lujn2r#L581):

```
../core/fxge/win32/cgdi_printer_driver.cpp(239,9): error: unknown type name 'WideString'; did you mean 'ByteString'?
  const WideString wsName =
        ^~~~~~~~~~
        ByteString
..\core/fxcrt/bytestring.h(278,7): note: 'ByteString' declared here
using ByteString = fxcrt::ByteString;
      ^
../core/fxge/win32/cgdi_printer_driver.cpp(240,7): error: use of undeclared identifier 'WideString'
      WideString::FromUTF8(pFont->GetFaceName().AsStringView());
      ^
```

It looks like a problem upstream.
This happens sometimes, they'll probably fix it soon.
Until then, you should download the previous build.

**Comment by mara004** [comment]
Okay, thanks for clarifying. I will then wait for the next release to support all three systems.

**Comment by mara004** [comment]
Again missing windows build today - please reopen

**Comment by bblanchon** [comment]
Already fixed in c2cf32bad3cac7f9fb3f57ac29739d4ffe0acc87.
Please download last week's build.
